src/address.js

```diff
@@ -38,8 +63,15 @@ function toOutputScript (address, network) {
   network = network || networks.bitcoin
 
   var decode = fromBase58Check(address)
+
   if (decode.version === network.pubKeyHash) return bscript.pubKeyHashOutput(decode.hash)
   if (decode.version === network.scriptHash) return bscript.scriptHashOutput(decode.hash)
+  if (decode.version === network.segWitPubKeyHash) {
+    if (decode.segWitVersion !== 0 && decode.segWitPadding !== 0) {
+      throw new TypeError(address + ' is too long for normal address and not a valid segWit address')
```

Comment:
not sure if this should go here or in `fromBase58Check`, considering the length checks are in `fromBase58Check`, this code has gotten a bit akward now ...

consider one of the test vectors for `fromBase58Check` we had of an address with hash length of 22, it no longer throws a 'is too long' exception and because we're unaware of `network` in `fromBase58Check` we can't really check if it's sane either ...

what was the original reason for seperating `fromBase58Check` and putting the exceptions there?